src/main/webapp/bower.json

```diff
@@ -1,25 +1,27 @@
 {
-  "name": "calendarPLUS",
-  "version": "0.0.0",
-  "homepage": "https://github.com/sylgas/IOSR2015_calendar",
-  "license": "MIT",
-  "ignore": [
-    "**/.*",
-    "node_modules",
-    "bower_components",
-    "test",
-    "tests"
-  ],
-  "dependencies": {
-    "angularjs": "1.3.15",
-    "angular-route": "1.3.15",
-    "bootstrap": "3.3.4",
-    "angular-ui-map": "0.5.0",
-    "angular-ui-calendar": "0.8.1",
-    "angular-bootstrap": "0.12.1"
-  },
-  "resolutions": {
-    "angular": "1.3.15",
-    "jquery": "2.1.4"
-  }
+    "name": "calendarPLUS",
+    "version": "0.0.0",
+    "homepage": "https://github.com/sylgas/IOSR2015_calendar",
+    "license": "MIT",
+    "ignore": [
+        "**/.*",
+        "node_modules",
+        "bower_components",
+        "test",
+        "tests"
+    ],
+    "dependencies": {
+        "angularjs": "1.3.15",
+        "angular-route": "1.3.15",
+        "bootstrap": "3.3.4",
+        "angular-ui-map": "0.5.0",
+        "angular-ui-calendar": "0.8.1",
+        "angular-bootstrap": "0.12.1",
+        "restangular": "1.5.1",
+        "lodash": "3.8.0"
```

Comment:
You don't need to add all that font-awesome files to repo. Just add proper line to dependencies here (e.g.  "fontawesome": "4.3.0", don't know which exactly version you have used) and add paths to proper files in index.html